graylog2-server/src/main/java/org/graylog2/migrations/V20180924111644_AddDefaultGrokPatterns.java

```diff
@@ -71,9 +72,19 @@ public void upgrade() {
                     .getResource("V20180924111644_AddDefaultGrokPatterns_Default_Grok_Patterns.json");
             final ContentPack contentPack = this.objectMapper.readValue(contentPackURL, ContentPack.class);
             final ContentPack pack = this.contentPackPersistenceService.insert(contentPack)
-                    .orElseThrow(() -> new Error("Content pack " + contentPack.id() + " with this revision " + contentPack.revision() + " already found!"));
+                    .orElseThrow(() -> {
+                        configService.write(MigrationCompleted.create(contentPack.id().toString()));
+                        return new ContentPackException("Content pack " + contentPack.id() + " with this revision " + contentPack.revision() + " already found!");
+                    });
 
-            contentPackService.installContentPack(pack, Collections.emptyMap(), "Add default Grok patterns", "admin");
+            try {
+                contentPackService.installContentPack(pack, Collections.emptyMap(), "Add default Grok patterns", "admin");
+            } catch(ContentPackException e) {
+                LOG.warn("Could not install default grok patterns: the installation found some modified default grok" +
+                        "patterns in your setup and did not update them. If you wish to use to the default grok" +
+                        "patterns we provide, please delete the grok pattern and install the 'Default grok" +
```

Comment:
Can we show which grok pattern caused troubles in there? Otherwise we need to rephrase this:
`(...) please delete the grok pattern (...)`